### PR TITLE
added colorAdd()

### PR DESCRIPTION
**Pull request by canTuncPekkan**
colorAdd() allows the user to add a new color with RGB

example;
colorAdd("Light Coral", (240, 128, 128))

now you can use "Light Coral" as a color

### CODE REVIEW

**Comment by canTuncPekkan** [comment]
umm I am new to GitHub and this is the first time this happened, why has it failed

**Comment by hugovk** [comment]
Thanks for the PR!

These are some automated checks to make sure tests pass and some code quality/linters are happy.

I added a changelog label, so that's now happy. There's still some lint things to check, but I'll add a review with some other suggestions first.

File: src/termcolor/termcolor.py [hunk 1]
```diff
@@ -126,6 +126,17 @@ def _can_do_colour(
     )
 
 
+def colorAdd(color: str, RGB: tuple | None = None) -> None:
+    """add colors to Available text colors
+
+    Args:
+        color (str): collor index name, used to call this collor
+        RGB (R, G, B): the RGB value of the collor you want to add. Defaults to None.
```
Comment (hugovk) [inline src/termcolor/termcolor.py:134]:
```suggestion
        color (str): color name
        rgb (R, G, B): the RGB value of the color you want to add. Defaults to None.
```

File: src/termcolor/termcolor.py [hunk 2]
```diff
@@ -126,6 +126,17 @@ def _can_do_colour(
...
+    """
+    if not RGB == None:
```
Comment (hugovk) [inline src/termcolor/termcolor.py:136]:
```suggestion
    if not rgb:
```

File: src/termcolor/termcolor.py [hunk 3]
```diff
@@ -140,7 +151,7 @@ def colored(
     Available text colors:
         black, red, green, yellow, blue, magenta, cyan, white,
         light_grey, dark_grey, light_red, light_green, light_yellow, light_blue,
-        light_magenta, light_cyan.
+        light_magenta, light_cyan, and all collors added by addCollor().
```
Comment (hugovk) [inline src/termcolor/termcolor.py:154]:
```suggestion
        light_magenta, light_cyan, and any colors added via add_color().
```

**Review by hugovk — CHANGES_REQUESTED** [review]
Please can you add some unit tests so we know this is working correctly?

**Comment by hugovk** [comment]
Reminder that this needs some updates. Let me know if you've any questions. Thanks!

**Comment by hugovk** [comment]
Closing due to inactivity, but do let me know if you'd like help and we can re-open.

**Pull request closed without merging**